core/node/block_reverter/Cargo.toml

```diff
@@ -14,12 +14,18 @@ zksync_types.workspace = true
 zksync_dal.workspace = true
 zksync_config.workspace = true
 zksync_contracts.workspace = true
+zksync_object_store.workspace = true
 zksync_storage.workspace = true
 zksync_eth_signer.workspace = true
 zksync_state.workspace = true
 zksync_merkle_tree.workspace = true
 
-tokio = { workspace = true, features = ["time"] }
-bitflags.workspace = true
+anyhow.workspace = true
+tokio = { workspace = true, features = ["time", "fs"] }
```

Comment:
seems `time` is not needed anymore

Reply:
This feature is still used [on this line](https://github.com/matter-labs/zksync-era/blob/fbd658652704d5bfba48e35873d5002a9280f9f4/core/node/block_reverter/src/lib.rs#L557).